cmd/main.go

```diff
@@ -53,6 +53,6 @@ func main() {
 	root.AddCommand(versionCommand)
 
 	if err := root.Execute(); err != nil {
-		panic(err)
+		os.Exit(1)
```

Comment:
The error is already printed out up top when an invalid flag is used and we figured that was enough information. We also think that `panic` gives the impression that something unhandled went wrong and a graceful exit with error code `1` creates less confusing output.